### ISSUE TITLE
add an example to the docs for transactions

### ISSUE THREAD

**Issue by lineus** [Automattic/mongoose#6672]
@vkarpov15 should we link to your existing blog post from the docs for an example of transactions or can we **steal** it :smile:

**Comment by sebastian-nowak** [comment]
Just wondering, why is `startSession` defined on the Model? I though that mongo supports multi collection transactions, so it feels a bit weird having to call it on a particular Model if a transaction is affecting multiple ones.

**Comment by vkarpov15** [comment]
Yeah it is a little strange. We did that because with Mongoose you generally work with models, not connections, and we wanted to make the API convenient as possible. We added `connection.startSession()` and `mongoose.startSession()` in #6689, those will be in 5.2.2. But `Model.startSession()` is just shorthand for `Model.db.startSession()`.

**Comment by florianwalther-private** [comment]
Does that mean I can use `Model.startSession` as a full replacement for `Mongoose.startSesssion`, even if I operate on multiple collections? The docs don't mention that.

**Comment by vkarpov15** [comment]
@florianwalther-private yes that is correct, `Model.startSession()` and `mongoose.startSession()` are equivalent if you define `Model` using `mongoose.model()`. Does that cause some issues for you?
